### DISCUSSION TITLE
Nextjs Auth with Meteor Server

### DISCUSSION THREAD

**Discussion by pisacode** [meteor/meteor#12432] [Feature Requests]
We are happily using meteor for 3 years but SSR and client performance is a continuing problem therefore we decided to create a nextjs app for client and would like to use our current Meteor app with Apollo Graphql Server as the backend.

We would like to integrate meteor users with nextj aut and I am wondering if anyone integrated these two stucture together or if meteor has a handler for this kind of actions since nextjs is now a go to solution for SSR

Thanks

**Comment by Grubba27** [comment]
For connecting both, we have a package suited for this, that is [simpleDDP](https://github.com/Gregivy/simpleddp), you can just plug in your nextJS app, and you will be able to connect fairly easily to Meteor.
I would use the DDP methods to do this kind of connection.
